### PR TITLE
Community creation of the Code of Conduct: Moved to PR #6 with contribution guideline

### PR DESCRIPTION
**Pull request by bvreede**
## Summary

* Opening a pull request with the explicit invite for members to respond to the code of conduct.
* While this pull request adds only a single minor change, please use this opportunity to respond to the entire Code of Conduct.
* Building on PR #3 and PR #4 by Malvika:

> ## Where can the project lead help
> * Please add a few members as designated CoC report taker and responders (just writing organisers won't cut it)
> * Use a dedicated email address for CoC reporting
> * Gather approval from the members as comments, or use emojis " 👍: for approval and 👎: disagree" on this PR
>   
> * CoC should not be a token, make people think about it intentionally
> * Do not close this PR without addressing these issue

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -72,7 +72,8 @@ disabilities.
 ### Reporting Issues
 
 If you experience or witness unacceptable behavior, or have any other concerns,
-please report it by contacting the organisers - <ADD EMAIL OF A DESIGNATED MEMBERS>.
+please report it by contacting the organisers:
+- <ADD EMAIL OF A DESIGNATED MEMBERS>.
```
Comment (ljgarcia) [inline CODE_OF_CONDUCT.md]: +1

**Comment by pjotrp** [comment]
@malvikasharan I have reverted the earlier PRs on the master branch, so we should have a clean slate. I am sorry I did that so fast - it is how I work (accept all PRs guided by ). As the PR system is not great anyway I'll invite everyone here as writers to the repo so you can merge your own changes.

**Comment by malvikasharan** [comment]
Thank you for re-opening this @bvreede ❤️️and @ljgarcia for responding 🌸 to a few points mentioned in the PR. 

@pjotrp Originally I would have opened an issue which is intended to get community members discussing various technical and organisational issues, but since that is not an option in your repo, and in GitHub that could be done over a PR, especially when marked with Work in Progress [WIP] - I assumed that you would read before merging them.

I have brought it up to Layla that since there is no defined guideline for contribution, it is not clear what is expected of the collaborators. I don't think I would participate in the event, but I will open another PR for [contribution guideline](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/), and I request you not to close that until all the members have clarified all the contribution pathways and agree to merge to the repository.

**Comment by pjotrp** [comment]
@malvikasharan thanks. Your efforts are appreciated. No worries about me making that mistake again.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+origin, political beliefs, profession, race, religion, sexual orientation,
+socioeconomic status, and technical ability. We will not tolerate discrimination
+based on any of the protected characteristics above, including participants with
+disabilities.
+
+### Reporting Issues
+
+If you experience or witness unacceptable behavior, or have any other concerns,
+please report it by contacting the organisers:
+- <ADD EMAIL OF A DESIGNATED MEMBERS>.
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md]:
```suggestion
Leyla Garcia (email: ljgarciaco@gmail.com), Pjotr Prins (email: pjotr.public708@thebird.nl) and Tazro Ohta (email: t.ohta@dbcls.rois.ac.jp).
```

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md]:
```suggestion
Leyla Garcia (email: ljgarciaco@gmail.com), Pjotr Prins (email: pjotr.public708@thebird.nl) and Tazro Ohta (email: t.ohta@dbcls.rois.ac.jp).
```

**Review by malvikasharan — COMMENTED** [review]
Adding names and emails.

**Review by malvikasharan — APPROVED** [review]

**Comment by malvikasharan** [comment]
I have opened a PR for the contribution guideline here: #6 

All the best!

**Pull request merged**